homeassistant/components/wemo/wemo_device.py

```diff
@@ -265,15 +276,15 @@ async def async_register_device(
     hass: HomeAssistant, config_entry: ConfigEntry, wemo: WeMoDevice
 ) -> DeviceCoordinator:
     """Register a device with home assistant and enable pywemo event callbacks."""
-    # Ensure proper communication with the device and get the initial state.
-    await hass.async_add_executor_job(wemo.get_state, True)
```

Comment:
There is already a test to ensure the device doesn't get created if the first update fails. To preserve that behavior, I made it raise `device.last_exception`